hasura/metadata/actions.graphql

```diff
@@ -66,6 +66,25 @@ type Mutation {
 }
 
 
+type Mutation {
+  create_offer (
+    limited: Boolean!
+    online_only: Boolean!
+    quantity: Int
+    offer_type: String!
+    description: String!
+    start_date: String
+    end_date: String
+    images: String!
+    sponsor_id: Int!
+    active: Boolean!
+    offer_name: String!
+    cost_in_tokens: Int!
+    icon: String!
+  ): create_offer_output
+}
+
+
```

Comment:
Remove space

Reply:
@steph222 we didn't add that double empty space, that is the Prettier tool I think